--- install.py~	2008-12-03 18:47:37.000000000 +0100
+++ install.py	2008-12-05 17:43:36.000000000 +0100
@@ -309,13 +309,13 @@
             install(src, dst)
         for src, link in MIME_LINKS:
             make_link(src, link)
-        os.popen('update-mime-database "%s"' % 
-            os.path.join(install_dir, 'share/mime'))
-        print '\nUpdated mime database (added .cbz, .cbr and .cbt file types.)'
-        schema = os.path.join(source_dir, 'mime/comicbook.schemas')
-        os.popen('GCONF_CONFIG_SOURCE=$(gconftool-2 --get-default-source) '
-                 'gconftool-2 --makefile-install-rule "%s" 2>/dev/null' %
-                    schema)
+#        os.popen('update-mime-database "%s"' % 
+#            os.path.join(install_dir, 'share/mime'))
+#        print '\nUpdated mime database (added .cbz, .cbr and .cbt file types.)'
+#        schema = os.path.join(source_dir, 'mime/comicbook.schemas')
+#        os.popen('GCONF_CONFIG_SOURCE=$(gconftool-2 --get-default-source) '
+#                 'gconftool-2 --makefile-install-rule "%s" 2>/dev/null' %
+#                    schema)
         print '\nRegistered comic archive thumbnailer in gconf (if available).'
         print 'The thumbnailer is only supported by some file managers,',
         print 'such as Nautilus'
